### PR TITLE
refactor: provide better explanation of offline-token login limits

### PR DESCRIPTION
**Pull request by craicoverflow**
**login help**
image

**cluster connect help**
image

**cluster connect interactive**
image

I've attached screenshots to aid visual verification.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation change
- [x] Other (please specify)

### Checklist

- [x] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: docs/commands/rhoas_cluster_connect.adoc [hunk 1]
```diff
@@ -40,8 +40,7 @@ $ rhoas cluster connect
       --ignore-context      Ignore currently selected services and ask to select each service separately
       --kubeconfig string   Location of the kubeconfig file.
   -n, --namespace string    Custom Kubernetes namespace (if not set current namespace will be used)
-      --token string        Provide an offline token to be used by the operator.
-                            To get a token, visit https://cloud.redhat.com/openshift/token.
+      --token string        Provide an offline token to be used by the operator. To obtain a token, visit https://cloud.redhat.com/openshift/token.
```
Comment (wtrocki) [inline docs/commands/rhoas_cluster_connect.adoc:43]:
```suggestion
      --token string        Provide an offline token to be used by the RHOAS operator. To obtain a token, visit https://cloud.redhat.com/openshift/token.
```

Comment (wtrocki) [reply docs/commands/rhoas_cluster_connect.adoc:43]: This should use constant

File: locales/cmd/login/active.en.toml [hunk 2]
```diff
@@ -12,6 +12,9 @@ one = '''
 Log in securely to RHOAS through a web browser.
 
 This command opens your web browser, where you can enter your credentials.
+
+Alternatively you can log in using an offline-token by passing the "--token" flag, which can be obtained at {{.OfflineTokenURL}}.
```
Comment (wtrocki) [inline locales/cmd/login/active.en.toml:16]:
```suggestion
For situatations when RHOAS CLI is used in automation without browser, you can log in using an offline-token by passing the "--token" flag, which can be obtained at {{.OfflineTokenURL}}.
```

File: locales/cmd/login/active.en.toml [hunk 3]
```diff
@@ -12,6 +12,9 @@ one = '''
...
+Note: token-based login is not supported by the Kafka "topic" and "consumergroup" subcommands.
```
Comment (wtrocki) [inline locales/cmd/login/active.en.toml:17]:
```suggestion
Note: token-based login is not supported by the Kafka "topic" and "consumergroup" subcommands and it is not recommended for personal use.
```

**Comment by wtrocki** [comment]
@craicoverflow sorry for taking this in draft state. Put some of recommendations we have mentioned

File: locales/cmd/login/active.en.toml (hunk 3 above)
Comment (craicoverflow) [reply locales/cmd/login/active.en.toml:17]: I'm unsure about highlighting that this is not recommended for personal use.

Comment (wtrocki) [reply locales/cmd/login/active.en.toml:17]: Sure. That is opiniated take of mine. Please ignore

File: docs/commands/rhoas_cluster_connect.adoc (hunk 1 above)
Comment (craicoverflow) [reply docs/commands/rhoas_cluster_connect.adoc:43]: You are reviewing the generated output, so it should be "https://cloud.redhat.com/openshift/token.".

File: locales/cmd/cluster/common/active.en.toml [hunk 4]
```diff
@@ -3,8 +3,7 @@ one = 'Location of the kubeconfig file.'
 
 [cluster.common.flag.offline.token.description]
 one = '''
-Provide an offline token to be used by the operator.
-To get a token, visit https://cloud.redhat.com/openshift/token.
+Provide an offline token to be used by the operator. To obtain a token, visit {{.OfflineTokenURL}}.
```
Comment (craicoverflow) [inline locales/cmd/cluster/common/active.en.toml:6]: Moved to a one-liner because the second line was not actually showing in the help of the command.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
@craicoverflow you need permission to merge that in code slush

**Comment by craicoverflow** [comment]
> @craicoverflow you need permission to merge that in code slush

Yes - I've requested 2 reviewers.

**Comment by craicoverflow** [comment]
merging this into #624 as they are closely tied.

**Pull request closed without merging**